### PR TITLE
[global] Cleanup old releases objects

### PR DESCRIPTION
**Pull request by yalosev**
## Description
Clean deprecated hooks and scripts

## Why do we need it, and what problem does it solve?
We don't need them anymore

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: global
type: chore
summary: Remove deprecated hooks and scripts
impact_level: default
```

### CODE REVIEW

**Review by shvgn — DISMISSED** [review]
For upmeter, LGTM

**Review by RomanenkoDenys — APPROVED** [review]
Reviewed bashible, LGTM

**Review by nabokihms — APPROVED** [review]
Prometheus ok.

File: ee/modules/110-istio/hooks/migration_remove_kiali_crds.go [hunk 1]
```diff
@@ -1,23 +0,0 @@
-/*
```
Comment (apolovov) [inline ee/modules/110-istio/hooks/migration_remove_kiali_crds.go:1]: Exclude this hook please, we are refactoring it in https://github.com/deckhouse/deckhouse/pull/2410.

**Review by apolovov — CHANGES_REQUESTED** [review]

File: ee/modules/110-istio/hooks/migration_remove_kiali_crds.go (hunk 1 above)
Comment (yalosev) [reply ee/modules/110-istio/hooks/migration_remove_kiali_crds.go]: Done

**Review by RomanenkoDenys — APPROVED** [review]

**Review by shvgn — APPROVED** [review]

**Review by apolovov — APPROVED** [review]

**Pull request merged**